backend/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
```

Comment:
don't push git ignore files

Reply:
Both the folders are different and require different package.json